xsd/netex_framework/netex_reusableComponents/netex_facility_support.xsd

```diff
@@ -245,6 +248,8 @@ Rail transport, Roads and Road transport
 			<xsd:enumeration value="panoramaCoach"/>
 			<xsd:enumeration value="pullmanCoach"/>
 			<xsd:enumeration value="standing"/>
+			<xsd:enumeration value="wheelchair"/>
```

Comment:
Can you move wheelchair below pushchair, like suitableForPushchairs, suitableForWheelchairs  etc.

Reply:
also agree on the better ordering

Reply:
Check. And check.